rosa_tutorials/rosa-mobb-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,336 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-aws-load-balancer-operator"]
+= AWS Load Balancer Operator on ROSA tutorial
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-aws-load-balancer-operator
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-12
+//---
+//date: '2023-01-03T22:07:08.574151'
+//title: AWS Load Balancer Operator On ROSA
+//aliases: ['/docs/rosa/alb-sts']
+//tags: ["AWS", "ROSA"]
+//authors:
+//  - Shaozhen Ding
+//  - Paul Czarkowski
+//---
+
+include::snippets/mobb-support-statement.adoc[leveloffset=+1]
+
+
+[TIP]
+====
+Load Balancers created by the AWS Load Balancer Operator (ALBO) cannot be used cannot be used for {product-title} Routes, and should only be used for individual services or Ingress that does not need the full layer 7 capabilties of a {product-title} route.
```

Comment:
Caught this while looking at this PR as a template/reference:

```suggestion
Load Balancers created by the AWS Load Balancer Operator (ALBO) cannot be used for {product-title} Routes, and should only be used for individual services or Ingress that does not need the full layer 7 capabilties of a {product-title} route.
```

Reply:
Thank you!